logicals.qmd

```diff
@@ -350,7 +350,7 @@ That leads us to the numeric summaries.
 
 When you use a logical vector in a numeric context, `TRUE` becomes 1 and `FALSE` becomes 0.
 This makes `sum()` and `mean()` very useful with logical vectors because `sum(x)` will give the number of `TRUE`s and `mean(x)` the proportion of `TRUE`s.
-That lets us see the distribution of delays across the days of the year as shown in @fig-prop-delayed-dist.
+Let's see the distribution of the proportion of delayed flights for each day of the year as shown in @fig-prop-delayed-dist.
```

Comment:
Please ignore my change to "That lets us". For some reason, I read it multiple times in my mind as "Then lets us... ". My apologies!

Reply:
```suggestion
That lets us see the distribution of delays across the days of the year as shown in @fig-prop-delayed-dist
```